example/spec/distributionAPI/distributionSpec.js

```diff
@@ -76,51 +97,109 @@ describe('Distribution API', () => {
     });
 
-    it('redirects to Earthdata login for unauthorized requests', async () => {
-      const response = await got(
-        fileUrl,
-        { followRedirect: false }
-      );
-      const authorizeUrl = new URL(response.headers.location);
-      expect(authorizeUrl.origin).toEqual(process.env.EARTHDATA_BASE_URL);
-      expect(authorizeUrl.searchParams.get('state')).toEqual(`/${config.bucket}/${fileKey}`);
-      expect(authorizeUrl.pathname).toEqual('/oauth/authorize');
-    });
+    describe('an unauthorized user', () => {
+      it('redirects to Earthdata login for unauthorized requests', async () => {
+        const response = await got(
+          fileUrl,
+          { followRedirect: false }
+        );
+        const authorizeUrl = new URL(response.headers.location);
+        expect(authorizeUrl.origin).toEqual(process.env.EARTHDATA_BASE_URL);
+        expect(authorizeUrl.searchParams.get('state')).toEqual(`/${protectedBucketName}/${fileKey}`);
+        expect(authorizeUrl.pathname).toEqual('/oauth/authorize');
+      });
 
-    it('redirecting to Earthdata login for unauthorized requests to /s3credentials endpoint.', async () => {
-      const response = await got(
-        `${process.env.DISTRIBUTION_ENDPOINT}/s3credentials`,
-        { followRedirect: false }
-      );
-      const authorizeUrl = new URL(response.headers.location);
-      expect(authorizeUrl.origin).toEqual(process.env.EARTHDATA_BASE_URL);
-      expect(authorizeUrl.searchParams.get('state')).toEqual('/s3credentials');
-      expect(authorizeUrl.pathname).toEqual('/oauth/authorize');
+      it('redirects to Earthdata login for requests on /s3credentials endpoint.', async () => {
+        const response = await got(
+          `${process.env.DISTRIBUTION_ENDPOINT}/s3credentials`,
+          { followRedirect: false }
+        );
+        const authorizeUrl = new URL(response.headers.location);
+        expect(authorizeUrl.origin).toEqual(process.env.EARTHDATA_BASE_URL);
+        expect(authorizeUrl.searchParams.get('state')).toEqual('/s3credentials');
+        expect(authorizeUrl.pathname).toEqual('/oauth/authorize');
+      });
+
+      it('downloads a public science file', async () => {
+        accessToken = randomId('accessToken');
+        const s3SignedUrl = await invokeLambdaForS3SignedUrl(publicFileUrl, accessToken);
+        const parts = new URL(s3SignedUrl);
+        const userName = parts.searchParams.get('x-EarthdataLoginUsername');
+
+        const fileStream = got.stream(s3SignedUrl);
+        const downloadChecksum = await generateChecksumFromStream('cksum', fileStream);
+        expect(userName).toEqual('publicAccess');
+        expect(downloadChecksum).toEqual(fileChecksum);
+      });
+
+      it('refuses downloads of files in private buckets (by redirecting for authentication)', async () => {
+        accessToken = randomId('accessToken');
+        const notASignedUrl = await invokeLambdaForS3SignedUrl(privateFileUrl, accessToken);
```

Comment:
It seems like the intent is to be testing that it redirects for authentication for unauthorized requests? You're passing an invalid access token, which has the same result, but seems like a different test case

Reply:
Really the intent was to show the AC `when an un-authenticated user tries to access private data via distribution url The user is denied access`

This could be explicit in middleware, but I hate that, but we don't refuse access to private-buckets until you're authenticated.  So you are totally correct that it's the same test with a different description.

Reply:
I guess the question is should I leave it or not.

Reply:
I guess I would remove it? To me, an "unauthenticated" user is someone who hasn't gone through any portion of the authentication process and thus shouldn't have an accessToken. As written, I think this test is more for `refuses downloads of files in private buckets when given invalid authentication`

Reply:
I see your point there with the random token.  Maybe I will change the AC to `when an unauthenticated user requests non-public data, they are redirected to authenticate`

Reply:
Updated the invokeDistributionApiLambda to be able to be called without a cookie and will move this.